src/sentry/static/sentry/app/views/organizationStats/index.jsx

```diff
@@ -267,7 +261,7 @@ const OrganizationStats = React.createClass({
                 <StackedBarChart
                   points={this.state.orgStats}
                   height={150}
-                  barClasses={['accepted', 'rate-limited', 'black-listed']}
+                  barClasses={['accepted', 'rate-limited']}
```

Comment:
Why are we dropping `black-listed` from the org stats bar chart? Why not have it there too?